scripts/src/java/org/oppia/android/scripts/todo/data/Issue.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.android.scripts.todo.data
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class that contains the required details of an [Issue] that is present in
+ * open_issues.json.
+ */
+@JsonClass(generateAdapter = true)
+data class Issue(
+  /** The issue number of the [Issue] */
+  @Json(name = "number") val issueNumber: String,
```

Comment:
Can this not be an Int? Does GitHub actually provide it as a string instead of an integer?